mirest/stylist-api
# #165378225 Configure flake8 to be run during testing on travis.

### What does this PR do?
Adds flake8 to the base application

#### Description of Task to be completed?
- Refactor files to follow pep8 after running flake8 command
- Add command to be run in Travis.

#### How should this be manually tested?

- Checkout to the branch `git checkout ch-configure-flake8-165378225`

- Run pipenv to install flake8

- Run `pipenv run flake8` to check for any pep8 errors.

#### What are the relevant pivotal tracker stories?
[#165378225](https://www.pivotaltracker.com/story/show/165378225)
